tests/common.rs

```diff
@@ -1,19 +1,25 @@
 #![allow(unused)]
+use std::io::Cursor;
 use std::net::Ipv4Addr;
 use std::ops::{Deref, DerefMut};
 use std::sync::Once;
 use std::time::{Duration, Instant};
 
+use pcap_file::pcap::PcapReader;
 use rand::Rng;
 use str0m::change::SdpApi;
 use str0m::format::Codec;
 use str0m::format::PayloadParams;
 use str0m::net::Receive;
+use str0m::rtp::ExtensionMap;
+use str0m::rtp::RtpHeader;
 use str0m::Candidate;
 use str0m::{Event, Input, Output, Rtc, RtcError};
 use tracing::info_span;
 use tracing::Span;
 
+const PCAP_VP8: &[u8] = include_bytes!("data/vp8.pcap");
```

Comment:
Including 69kb of data in the test binary seems ill-advised. It's useful to include binary data like this when you want a single binary, for tests we don't care and could just read the file from disk in the test instead, in particular this might slow down test compilation which would be a problem

Reply:
true if we include many for more tests

Reply:
I don't believe it makes any difference whatsoever, since the compilation unit (common.rs) will be compiled once. But also I don't care either way.